docker-compose/db/createSubjects.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#
+# create the subjects databases (in Cloudant or CouchDB)
+#
+# Usage: createSubjects.sh
+
+SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
+
+URL_BASE="$OPEN_WHISK_DB_PROTOCOL://$OPEN_WHISK_DB_HOST:$OPEN_WHISK_DB_PORT"
+
+if [ "$OPEN_WHISK_DB_PROVIDER" == "CouchDB" ]; then
+    CURL_ADMIN="curl -s -k --user $OPEN_WHISK_DB_USERNAME:$OPEN_WHISK_DB_PASSWORD"
```

Comment:
when this failed for me, i had to edit the curl to add `-v` to see why it failed (couldn't reach host).